lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -960,8 +975,6 @@ sub update_status {
         }
     }
     $self->update();
-    # mark the worker as alive
-    $self->worker->seen;
```

Comment:
well, I didn't follow the complete implementation, e.g. now it's called after update, but anway, go ahead if you are feeling secure … remembering that our test coverage for everything worker related is not so good ;-)